proto/raft.proto

```diff
@@ -32,6 +32,7 @@ message Isolation {
 message FileRecord {
   Isolation isolation = 1;
   build.bazel.remote.execution.v2.Digest digest = 2;
+  int64 size_bytes = 3;
```

Comment:
Could we put this in FileMetadata instead?

(otherwise it's a little confusing whether or not to look at digest.GetSizeBytes() or fileRecord.GetSizeBytes())

Reply:
Done.